avm/ptn/data/private-analytical-workspace/tests/common.tests.ps1

```diff
@@ -213,12 +213,12 @@ function Test-VerifyKeyVault($KeyVaultResourceGroupName, $KeyVaultName, $Tags, $
     $kv.EnableRbacAuthorization | Should -Be $true
     $kv.EnableSoftDelete | Should -Be $EnableSoftDelete
     $kv.SoftDeleteRetentionInDays | Should -Be $RetentionInDays
-    $kv.EnablePurgeProtection | Should -Be $true
+    $kv.EnablePurgeProtection | Should -BeIn @($false, $null)
     $kv.PublicNetworkAccess | Should -Be $PublicNetworkAccess
     $kv.AccessPolicies | Should -BeNullOrEmpty
     $kv.NetworkAcls.DefaultAction | Should -Be 'Deny'
     $kv.NetworkAcls.Bypass | Should -Be 'None'
-    if ( $IpAddressRanges -eq $null ) { $kv.NetworkAcls.IpAddressRanges | Should -BeNullOrEmpty }
+    if ( $null -eq $IpAddressRanges ) { $kv.NetworkAcls.IpAddressRanges | Should -BeNullOrEmpty }
```

Comment:
Why this change?

Reply:
Because of 2 reasons:
1. It's a bad practice and other, less forgiving programming language can fail this check with a NullPointerException if you don't put the null-check to the left of the equation
2. The Script Analyzer highlighted the same with some very triggering yellow scribberly lines that I just had to address. Trust me, there was simply no other way

cc: @eriqua

Reply:
Learn something new every day, thanks! 
https://learn.microsoft.com/en-us/powershell/scripting/learn/deep-dives/everything-about-null?view=powershell-7.4#checking-for-null